feathr_project/feathr/client.py

```diff
@@ -442,3 +442,7 @@ def _get_sql_config_str(self):
             JDBC_PASSWORD: "{JDBC_PASSWORD}"
             """.format(JDBC_TABLE=table, JDBC_USER=user, JDBC_PASSWORD=password)
         return config_str
+
+    def get_features_from_registry(self, project_name):
+        """ Sync features from the registry given a project name """      
+        self.registry.get_features_from_registry(project_name, os.path.abspath("./"))
```

Comment:
Is this by design that specify workspace path is not exposed to user? They can only achieve so, by execute code under certain workspace folder.

Reply:
Added TODO to address this